### ISSUE TITLE
About number of views as input to dinov2

### ISSUE THREAD

**Issue by LeeCASC** [microsoft/TRELLIS#80]
How many multi-view rendering images do you use for training?

**Comment by FishWoWater** [comment]
According to Section 4 in the paper, maybe 150 images per asset.

**Comment by LeeCASC** [comment]
> According to Section 4 in the paper, maybe 150 images per asset.

**Comment by LeeCASC** [comment]
> According to Section 4 in the paper, maybe 150 images per asset.
I also saw in the paper, but 150 is the total number of 3D shape renderings. I want to know how many 2D rendering images are used in each iteration.

**Comment by YuDeng** [comment]
To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step.

**Comment by LeeCASC** [comment]
> To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> 
> Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).
Thanks for your reply~~Further questions,
Is the process of using dinov2 to obtain 150 view features frozen?
Is the process of voxelize frozen?

**Comment by YuDeng** [comment]
> > To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> > Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).
> > Thanks for your reply~~Further questions,
> > Is the process of using dinov2 to obtain 150 view features frozen?
> > Is the process of voxelize frozen?

The dinov2 and the voxelization process are all fixed. No gradient is needed.

**Comment by LeeCASC** [comment]
> To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> 
> Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).

Thank you for your reply~
In each step, randomly sample only one rendering image from a certain perspective as supervision for VAE?
Is the input of VAE only a fusion representation of voxelization and multi-view features?

**Comment by YuDeng** [comment]
> > To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> > Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).
> 
> Thank you for your reply~ In each step, randomly sample only one rendering image from a certain perspective as supervision for VAE? Is the input of VAE only a fusion representation of voxelization and multi-view features?

Yes, we sample one random view out of the rendered 150 views of the object.

The input to VAE is just a sparse grid of 64^3, and each voxel stores the average Dino-v2 feature from the 150 views.

**Comment by LeeCASC** [comment]
> > > To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> > > Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).
> > 
> > 
> > Thank you for your reply~ In each step, randomly sample only one rendering image from a certain perspective as supervision for VAE? Is the input of VAE only a fusion representation of voxelization and multi-view features?
> 
> Yes, we sample one random view out of the rendered 150 views of the object.
> 
> The input to VAE is just a sparse grid of 64^3, and each voxel stores the average Dino-v2 feature from the 150 views.

Thank you very much~~Best wishes for you!

**Comment by LeeCASC** [comment]
> > > To obtain SLAT, we aggregate 150 views. During training, for loss computation of the VAE, we randomly sample one rendered image per object at each step, with a batchsize of 256 of randomly sampled objects.
> > > Also see [#52](https://github.com/microsoft/TRELLIS/issues/52).
> > 
> > 
> > Thank you for your reply~ In each step, randomly sample only one rendering image from a certain perspective as supervision for VAE? Is the input of VAE only a fusion representation of voxelization and multi-view features?
> 
> Yes, we sample one random view out of the rendered 150 views of the object.
> 
> The input to VAE is just a sparse grid of 64^3, and each voxel stores the average Dino-v2 feature from the 150 views.

Both 3D asset voxelization and multi-view image feature extraction use pre-trained models. Did you train the 3D asset voxelization pre-trained model yourself?
